source/fundamentals/linq.txt

```diff
@@ -301,6 +339,44 @@ returns the rest. The result contains the following documents:
    { "name" : "Forlinis Restaurant", "cuisine" : "Italian" }
    ...
 
+.. _csharp-linq-skipwhile:
+
+Skip on a Condition
+```````````````````
+
+You can use the ``SkipWhile()`` LINQ method in a ``Select()`` projection
+to skip array field elements while a specified condition is true, then
+return the remaining elements.
+
+This example uses the following ``Student`` class to model documents
+that contain an array field ``Grades``:
+
+.. code-block:: csharp
+
+   public class Student
+   {
+       public ObjectId Id { get; set; }
+       public string Name { get; set; }
+       public int[] Grades { get; set; }
+   }
+
+The following code shows how to use the ``SkipWhile()`` method to skip
+any ``Grades`` array elements that are less than ``75`` and return the
+rest of the array:
+
+.. code-block:: csharp
+
+   var query = queryableCollection
+       .Select(s => s.Grades.SkipWhile(g => g < 75).ToArray());
+
+The results might resemble the following documents:
```

Comment:
The results look right, but calling them "documents" is not quite right.

Maybe just say "The results might resemble the following:"